images/scorecard-proxy/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Binary builder image
+FROM golang:1.10-alpine3.8 AS builder
```

Comment:
Same problem as #431: docker support on RHEL

Reply:
This is intended to be built in CI and pushed to quay.io. Since we don't expect users to build the proxy, I don't think we need to worry about RHEL support as long as the CI supports it.